templates/swift/.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+  "recommendations": [
+    "dtsvet.vscode-wasm"
```

Comment:
What does this provide?

Also consider recommending the LLDB extension since you use that in the debug actions.

Reply:
Oh -- that is a good point. This one provides some tools for syntax highlighting and inspecting WAT and WASM files. I should see if it actually adds value to the average WASM dev. I use it frequently... but that's because I tend to tinker with low level stuff.